src/edu/washington/escience/myria/CsvTupleWriter.java

```diff
@@ -79,4 +95,14 @@ public void error() throws IOException {
       csvPrinter.close();
     }
   }
+
+  private void writeBBtoFile(final ByteBuffer bb, final String fn) throws IOException {
+    boolean append = false;
```

Comment:
Inlined append. I need either a randomaccess file or a file output stream, file output stream seems to be the most common pattern, so I am using that.